.github/workflows/pr.yaml

```diff
@@ -10,12 +10,10 @@ jobs:
     steps:
       - name: check out pr branch
         uses: actions/checkout@v2
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
       - name: Build test container
         env:
           DOCKER_BUILDKIT: 1
-        run: docker build -t civiform --cache-from docker.io/civiform/civiform:latest ./
+        run: DOCKER_BUILDKIT=1 docker build -t civiform ./
```

Comment:
this would appear to be redundant with the env right above... Is this WAI and just unfamiliar to me or odd enough we should comment it?

Reply:
done